JavaScript2/readme.md

```diff
@@ -0,0 +1,7 @@
+## Here you can find course content and homework for the JavaScript 2 module   
+
+| Week | Topic | Preparation | Homework | Review |
+| ---- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | ------------------------------------------------------------------------ |
+| 4.   | • JSON <br>• Array Manipulation <br>• Structuring code files | [Preparation](/JavaScript2/Week4/preparation.md)                                                                                  | [Homework](/JavaScript2/Week4/homework.md)                | [Review](/JavaScript2/Week4/review.md)                                               |
+| 5.   | • First Git Session<br>• Events<br>• Callbacks <br>• XHTTP Requests <br>• API calls                                                                                                           | [Preparation](/JavaScript2/Week5/preparation.md)                                                                                  | [Homework](/JavaScript2/Week5/homework.md)   | [Review](/JavaScript2/Week5/review.md)                                                                   |
```

Comment:
Are we doing another first and second GIT session though?

Reply:
No, well spotted